README.md

```diff
@@ -2,7 +2,191 @@
+  Quine's "New Foundations" (NF) set theory axioms, a direct derivative
+  of the "hierarchy of types" set theory originally presented in
+  Whitehead and Russell's *Principia Mathematica*.
+* "ql.mm" aka "Quantum Logic Explorer" - Starts from the orthomodular
+  lattice properties proved in the Hilbert Space Explorer and takes
+  you into quantum logic.
+* "hol.mm" aka "Higher-Order Logic Explorer" - Starts with HOL (also
+  called simple type theory) and derives equivalents to ZFC axioms,
+  connecting the two approaches.
+* "peano.mm" - Peano arithmetic.
+* "big-unifier.mm" - a unification and substitution test for
+  Metamath verifiers, where small input expressions blow up to thousands
+  of symbols.  It is a translation of William McCune's "big-unifier.in"
+  for the OTTER theorem prover.
+* "miu.mm"  - a simple formal system for use as a demonstration based
+  on work by Hofstadter.
+* "demo0.mm" - a simple formal system used as a demonstration in
+  Chapter 2 of the Metamath book.
+
+## How are they verified?
+
+Changes ("commits") to any database are first automatically verified
+before they are accepted, using GitHub actions.
+
+The set.mm and iset.mm databases are the ones primarily being updated.
+Every change to these databases are verified by the following
+different verifiers:
+
+* metamath.exe aka Cmetamath (the original C verifier by Norman Megill)
+* checkmm (a C++ verifier by Eric Schmidt)
+* smetamath-rs (smm3) (a Rust verifier by Stefan O'Rear)
+* mmj2 (a Java verifier by Mel L. O'Cat and Mario Carneiro)
+* mmverify.py (a Python verifier by Raph Levien)
+
+Note that these are different verifiers written in different programming
+languages by different people. In addition, the verification algorithm
+is intentionally simple (it fits in two pages in the Matamath book).
+This provides *extremely* high confidence that the proofs are correct.
+Most math proofs are not formally verified at all (that is, where a
+computer verification verifies every step). The rest are typically only
+verified by a single program (which might have an error in it).
+These multiple independent verifications provide very high confidence
+of their being correct.
+
+All other databases are verified by at least one verifier.
+
+## How can I contribute?
+
+First, take a look at some existing proofs.
+Walking through the 
+[Metamath Proof Explorer (MPE, aka set.mm)](http://us.metamath.org/mpegif/mmset.html) is a good way to start.
+
+To create a proof, you need to pick a tool.
+The Metamath system specifies a common file format, not a single tool
+everyone has to use.
+Many in the community use the [mmj2 tool](http://us.metamath.org#book).
+The video [Introduction to Metamath and mmj2](https://www.youtube.com/watch?v=Rst2hZpWUbU) demonstrates using mmj2.
+The mmj2 tool includes an interactive tutorial, and if you just want to watch,
+there's a video giving a
+[Walkthrough of the tutorial in mmj2](https://www.youtube.com/watch?v=87mnU1ckbI0&t=2094s).
```

Comment:
Do you mean to link to the video starting at a timestamp halfway through the video?

Reply:
You're right, thanks for noticing! Fixed in the version I'm about to post.